canu/inventory/plays/aruba-aoscx.yml

```diff
@@ -104,35 +105,43 @@
           ansible.builtin.set_fact:
             mgmt_int: "{{ running_config_data.json.System.mgmt_intf }}"
 
+        # subsystem config is needed for the 6300s.  Otherwise when we upload the config the switch will wipe itself.
```

Comment:
what does this do w/ 8360's?  if it's accidentally done on a spine or leaf does it hurt or mismatch anything?

Reply:
I ran it on Odin which has 8325 48p, 8325 32p, & 8360 and it didn't cause any issues. 

It preserves this config on 8360
```json
        "subsystems": {
            "chassis/1": {
                "name": "1",
                "type": "chassis"
            },
            "fan_tray/1%2F1": {
                "name": "1/1",
                "type": "fan_tray"
            },
            "fan_tray/1%2F2": {
                "name": "1/2",
                "type": "fan_tray"
            },
            "fan_tray/1%2F3": {
                "name": "1/3",
                "type": "fan_tray"
            },
            "line_card/1%2F1": {
                "name": "1/1",
                "part_number_cfg": "JL720A",
                "type": "line_card"
            },
            "management_module/1%2F1": {
                "name": "1/1",
                "type": "management_module"
            }
        },
```

8325
```json
        "subsystems": {
            "chassis/1": {
                "name": "1",
                "type": "chassis"
            },
            "line_card/1%2F1": {
                "name": "1/1",
                "part_number_cfg": "JL636A",
                "type": "line_card"
            },
            "management_module/1%2F1": {
                "name": "1/1",
                "type": "management_module"
            }
        },
```